domain/build/outputs/logs/manifest-merger-debug-report.txt

```diff
@@ -0,0 +1,37 @@
+-- Merging decision tree log ---
```

Comment:
Ditto for this file--let's remove it if we don't need it.

Reply:
deleted